bower.json

```diff
@@ -25,7 +25,8 @@
     "angular-hotkeys": "chieffancypants/angular-hotkeys#~1.4.3",
     "favico.js": "~0.3.5",
     "seedrandom": "~2.3.11",
-    "decimal.js": "git://github.com/MikeMcl/decimal.js.git#~4.0.1"
+    "decimal.js": "git://github.com/MikeMcl/decimal.js.git#~4.0.1",
+    "dropbox-datastores-1.0-latest": "https://www.dropbox.com/static/api/dropbox-datastores-1.0-latest.js"
```

Comment:
(this is fine as is - I'd hoped http://bower.io/search/?q=dropbox would have this nicely packaged, but there's nothing widely used there. "Dropbox core API" is there but it's not the same thing. This comment's just so I don't forget.)